autoload/cmp_nvim_ultisnips.vim

```diff
@@ -1,73 +1,3 @@
-" TODO: move python code into separate files
-
-" Retrieves additional snippet information that is not directly accessible
-" using the UltiSnips API functions. Returns a list of tables (one table
-" per snippet) with the keys "trigger", "description", "options" and "value".
-"
-" If 'expandable_only' is "True", only expandable snippets are returned, otherwise all
-" snippets except regex and custom context snippets for the current filetype are returned.
-function! cmp_nvim_ultisnips#get_current_snippets(expandable_only)
-let g:_cmpu_current_snippets = []
-python3 << EOF
-import vim
-from UltiSnips import UltiSnips_Manager, vim_helper
-
-before = vim_helper.buf.line_till_cursor
-visual_content = UltiSnips_Manager._visual_content
-expandable_only = vim.eval("a:expandable_only") == "True"
-if expandable_only:
-    snippets = UltiSnips_Manager._snips(before, True)
-else:
-    snippets = UltiSnips_Manager._snips("", True)
-
-for snippet in snippets:
-    is_context_snippet = snippet._context_code != None
```

Comment:
Shall we still keep these in the new python file? I saw from line 24 - 32 is missed in the new python file

Reply:
Thanks, I missed that